docs/source/ecf.rst

```diff
@@ -865,17 +865,22 @@ run_myfuncs
 '''''''''''
 Determines the astrophysical and systematics models used in the Stage 5 fitting.
 For standard numpy functions, this can be one or more (separated by commas) of the following:
-[batman_tr, batman_ecl, sinusoid_pc, expramp, hstramp, polynomial, step, xpos, ypos, xwidth, ywidth, GP].
+[batman_tr, batman_ecl, poet_tr, poet_ecl, sinusoid_pc, expramp, hstramp, polynomial, step, xpos, ypos, xwidth, ywidth, lorentzian, damped_osc, GP].
 For theano-based differentiable functions, this can be one or more of the following:
 [starry, sinusoid_pc, expramp, hstramp, polynomial, step, xpos, ypos, xwidth, ywidth],
 where starry replaces both the batman_tr and batman_ecl models and offers a more complicated phase variation model than sinusoid_pc that accounts for eclipse mapping signals.
+The POET transit and eclipse models are best-suited for planets with small eccentricities.  POET has a fast implementation of the 4-parameter limb darkening model that is valid for small planets (Rp/Rs < 0.1)
```

Comment:
I added text indicating that POET assumes a symmetric transit shape.  It's really up to the user to decide what "small" means, but I've added my two cents (e < 0.2).

My plan is to eventually add POET's C library to Eureka!.  This will replace the functions near the bottom of PoetModels.py.  Given that my python cose is already comparable in speed to BATMAN, the speed boost with POET's C library could be quite substantial, assuming that's where the bottleneck is in S5.

Reply:
We need to do some code profilling on S5.

Reply:
In the interim, you might consider using numba's `@jit` decorator (https://numba.readthedocs.io/en/stable/user/jit.html) which requires a small amount of compile time at startup but can speed up frequently called functions with basically no development cost

Reply:
Applying `jit` to all of the functions called frequently in S5 is a good idea for another PR.